platform_login/index.js

```diff
@@ -52,7 +52,7 @@ module.exports.handler = (event, context, callback) => {
 			}
 
 			var authenticationData = {
-				Username : event.body.username,
+				Username : event.body.username.toLowerCase(),
```

Comment:
I hope this is not a change related to CAPI-1252

Reply:
This is not a change related to CAPI-1252